package.json

```diff
@@ -35,7 +35,7 @@
   },
   "scripts": {
     "start": "npm run storybook",
-    "storybook": "start-storybook -p 9001 --stories \"./packages/*/stories/*.stories.{js,mdx}\" --node-resolve --watch --open",
+    "storybook": "start-storybook -p 9001 --stories \"./packages/calendar/stories/*.stories.{js,mdx}\" --node-resolve --watch --open",
```

Comment:
Should we merge with `calendar` being hardcoded or should `*` be put back?

Reply:
We're merging these to next/storybook. Once everything is there, we can set it to * again. If we do it before, storybook doesn't work because of combining .js with old format, with .mdx story files. So for reviewing purposes I left it at calendar so you can just run npm start and review the storybook ;)